src/sagemaker/jumpstart/factory/model.py

```diff
@@ -643,13 +640,9 @@ def _add_config_name_to_deploy_kwargs(
             specs=specs, training_config_name=training_config_name
         )
 
-    if (
-        specs.inference_configs
-        and specs.inference_configs.get_top_config_from_ranking().config_name
-    ):
-        kwargs.config_name = (
-            kwargs.config_name or specs.inference_configs.get_top_config_from_ranking().config_name
-        )
+    if specs.inference_configs and specs.inference_configs.get_top_config_from_ranking():
```

Comment:
it seems you just got rid of `.config_name` from the if-statement

Reply:
sorry extracted to `default_config_name` and removed the redundant check `..get_top_config_from_ranking()` in the if clause. This is not needed since it fallbacks to None anyway